code/modules/fishing/sources/_fish_source.dm

```diff
@@ -57,6 +57,11 @@ GLOBAL_LIST_INIT(specific_fish_icons, zebra_typecacheof(list(
 	var/explosive_malus = FALSE
 	/// If explosive_malus is true, this will be used to keep track of the turfs where an explosion happened for when we'll spawn the loot.
 	var/list/exploded_turfs
+	/// Mindless mobs that can fish will never pull up items on this list
+	var/static/list/profound_fisher_blacklist = typecacheof(list(
+		/mob/living/basic/mining/lobstrosity,
```

Comment:
Didn't you add baby lobsters to the fishing list too, or do you not mind if they fish those up? Will it not lead to similar issues?

Reply:
It will lead to similar issues because baby lobstrosities also fish.

Reply:
I'm dumb and forgot that this already includes them on account of being a subtype